Automattic/themes
# Seedlet Blocks: bring up-to-date with GB 10.3

This PR cherry picks the work to make Seedlet Blocks a child of BCB, so we can deploy it and iterate separate from Mayland Blocks.

If it looks okay, I'll squash merge and deploy to wpcom.

**Testing instructions**
- Use the horizon link on wordpress.com with gutenboarding + site editor flags to create a new site
- Check out this PR onto your sandbox and sandbox the site you just created
- Activate seedlet blocks using the CLI (`wp theme activate pub/seedlet-blocks`)
- Go to the site editor and verify it looks right
